src/client/cypress/e2e/detailPage/log.cy.js

```diff
@@ -65,4 +66,22 @@ describe("Test for the borehole log.", () => {
     verifyFullRowContent(["R96", "10.0 - 20.0", "CAL, GYRO, GR", "Other"], 0);
     verifyRowWithTextCheckState("R96", true);
   });
+
+  it("filters log runs in table", () => {
+    // Todo:  temporarily test with existing borehole, adding logs is not yet implemented. Integrate in new test when implemented.
+    goToDetailRouteAndAcceptTerms(`/1000070/log?dev=true`);
+    assertCountDisplayed("10 runs");
+    getElementByDataCy("filter-button").should("exist");
+    getElementByDataCy("filter-form").should("not.exist");
+    getElementByDataCy("filter-button").click();
+    getElementByDataCy("filter-form").should("exist");
+    toggleMultiSelect("runNumbers", [3], 11); // "R49
+    assertCountDisplayed("1 run");
+    setSelect("section", 3, 8); // "Belgium (54.0 - 141.0)"
+    assertCountDisplayed("0 runs");
+    toggleMultiSelect("runNumbers", [3], 11);
+    assertCountDisplayed("5 runs");
+    toggleMultiSelect("toolTypes", [2, 3]);
+    assertCountDisplayed("2 runs");
```

Comment:
Q: Gibt es die reset option in den Selects? Falls ja PP: Testcase zum Reset darüber hinzufügen.